### DISCUSSION TITLE
Parse Ethernet data logged in .mf4 file

### DISCUSSION THREAD

**Discussion by Bhargav-KM** [danielhrisca/asammdf#864] [Q&A]
Hi, is there any way to extract/parse **ethernet** .mf4 ?

I am trying but not extracting data.

file = MDF(ethernet_mf4)
print(file.attachments)

output:
[ATBLOCK(address=340, file_name=VectorBlfMetadata1.xml, comment=Vector Blf Metadata with general information),
 ATBLOCK(address=580, file_name=VectorBlfMetadata2.xml, comment=Vector Blf Metadata with channel information),
 ATBLOCK(address=7f0, file_name=VectorBlfMetadata3.xml, comment=Vector Blf Metadata with identity information)]

In the above output there is no ethernet arxml database file attached, so is that the reason for MDF not extracting ethernet data?

**Comment by danielhrisca** [comment]
Even with the arxml there is no decoding implemented for ethernet frames. It would be really useful if you could share a demo mf4 and arxml (see profile email address)
